src/pyvcam_driver.py

```diff
@@ -0,0 +1,537 @@
+
+        :param value: Desired binning value as a tuple, or int for a square binning.
+
+        :return: None.
+        """
+        self.cam.binning = value
+    
+    def chip_name(self) -> str:
+        """
+        Returns the camera sensor's name as a string.
+
+        :return:
+        """
+        return self.cam.chip_name
+    
+    def post_trigger_delay(self) -> int:
+        """
+        Returns the last acquisition's post-trigger delay reported by the camera in ms.
+        
+        :return:
+        """
+        return self.cam.post_trigger_delay
+    
+    def pre_trigger_delay(self) -> int:
+        """
+        Returns the last acquisition's pre-trigger delay reported by the camera in ms.
+
+        :return:
+        """
+        return self.cam.pre_trigger_delay
+    
+    def scan_line_time(self) -> int:
+        """
+        Returns the scan line time in ns.
+
+        :return:
+        """
+        return self.cam.scan_line_time
+    
+    def sensor_size(self) -> tuple[int, int]:
+        """
+        Returns the sensor size of the current camera in a tuple 
+        in the form (serial sensor size, parallel sensor size)
+
+        :return:
+        """
+        return self.cam.sensor_size
+    
+    def speed_table_index(self) -> int:
+        """
+        Returns the current numerical index of the speed table of the camera.
+        """
+        return self.cam.speed_table_index
+    
+    def set_speed_table_index(self, value: int) -> None:
+        """
+        Changes the current numerical index of the speed table of the camera. 
+        See the Port and Speed Choices section inside the PVCAM User Manual for 
+        a detailed explanation about PVCAM speed tables.
+
+        :param value: Desired speed table index.
+
+        :return: None.
+        """
+        self.cam.speed_table_index = value
+    
+    def temp(self) -> float:
+        """
+        Returns the current temperature of a camera in Celsius.
+
+        :return:
+        """
+        return self.cam.temp
+    
+    def temp_setpoint(self) -> float:
+        """
+        Returns the camera's temperature setpoint. The temperature setpoint is 
+        the temperature that a camera will attempt to keep it's temperature (in Celsius) at.
+
+        :return:
+        """
+        return self.cam.temp_setpoint
+    
+    def set_temp_setpoint(self, value: int) -> None:
+        """
+        Changes the camera's temperature setpoint.
+
+        :return: None.
+        """
+        self.cam.temp_setpoint = value
+    
+    def trigger_table(self) -> dict[str, str]:
+        """
+        Returns a dictionary containing a table consisting of information of the last acquisition such as exposure time, 
+        readout time, clear time, pre-trigger delay, and post-trigger delay. 
+        If any of the parameters are unavailable, the dictionary item will be set to 'N/A'.
+
+        :return:
+        """
+        return self.cam.trigger_table
```

Comment:
```suggestion
        return self.cam.trigger_table
 
```
Add terminating newline.